docs/dev/Lua API.rst

```diff
@@ -5348,11 +5348,15 @@ If the panel has already been maximized in this fashion, then it will jump to
 its minimum size. Both jumps respect the resizable edges defined by the
 ``resize_anchors`` attribute.
 
-The time duration that a double click can span is defined by the global variable
-``DOUBLE_CLICK_MS``. The default value is ``500`` and can be changed by the end
+The time duration that a double click can span can be controlled by widgets API:
```

Comment:
Could you update this to indicate that the `control-panel` or `gui/control-panel` interfaces should be used to set these properties? That goes for the others as well. You can remove the `widget.*` function references. These docs were written before the control panel was a thing.

The `gui/control-panel` interface looks like this:
![image](https://github.com/user-attachments/assets/2ce3195e-b397-450a-b050-e28cf7593368)

and the `control-panel` interface looks like this:
![image](https://github.com/user-attachments/assets/22b41f5b-32ec-457e-a2f3-754171eaa100)

the functions should no longer be called by anything other than the control panel interfaces